src/Compilers/Core/Portable/Symbols/TypeCompareKind.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.CodeAnalysis
     [Flags]
     internal enum TypeCompareKind
     {
-        ConsiderEverything = 0, // Perhaps rename since this does not consider nullable modifiers.
+        ConsiderEverything = 0,
```

Comment:
>ConsiderEverything [](start = 8, length = 18)

Consider leaving a comment or xml doc to indicate this option does not compare nullability.

Reply:
> Consider leaving a comment or xml doc to indicate this option does not compare nullability.

But it does, the comment was obsolete.

---
In reply to: [230518210](https://github.com/dotnet/roslyn/pull/30913#discussion_r230518210) [](ancestors = 230518210)